astropy/modeling/core.py

```diff
@@ -1110,17 +1110,16 @@ def render(self, out=None, coords=None):
             # Check dimensions match out and model
             assert len(coords) == ndim
             if out is not None:
-                assert coords[0].shape == out.shape
+                if coords[0].shape != out.shape:
+                    raise AssertionError
```

Comment:
Change this one to `ValueError('inconsistent shape of the output')`? Or does that fail tests?